packages/blade/src/components/Popover/__tests__/Popover.web.test.tsx

```diff
@@ -0,0 +1,281 @@
+/* eslint-disable @typescript-eslint/restrict-plus-operands */
+/* eslint-disable @typescript-eslint/require-await */
+/* eslint-disable @typescript-eslint/no-empty-function */
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
+import { act, fireEvent } from '@testing-library/react';
+import React from 'react';
+import { Popover, PopoverInteractiveWrapper } from '..';
+import { Button } from '~components/Button';
+import { paymentTheme } from '~tokens/theme';
+import { InfoIcon } from '~components/Icons';
+import { MetaConstants } from '~utils/metaAttribute';
+import renderWithTheme from '~utils/testing/renderWithTheme.web';
+import assertAccessible from '~utils/testing/assertAccessible.web';
+import { Box } from '~components/Box';
+import { Text } from '~components/Typography';
+
+const waitForPosition = () => act(async () => {});
+const animationDuration = paymentTheme.motion.duration.quick;
+
+describe('<Popover />', () => {
+  jest.useFakeTimers();
+
+  it('should render', () => {
+    const buttonText = 'Click me';
+    const { container, getByRole, queryByRole } = renderWithTheme(
+      <Popover content="Hello world">
+        <Button>{buttonText}</Button>
+      </Popover>,
+    );
+
+    // snapshot while on opened
+    fireEvent.click(getByRole('button', { name: buttonText }));
+    expect(queryByRole('dialog')).toBeInTheDocument();
+    expect(queryByRole('dialog')).toHaveStyle({ 'z-index': 1100 });
+    expect(queryByRole('dialog')).toHaveAttribute('data-blade-component', MetaConstants.Popover);
+    expect(container).toMatchSnapshot();
```

Comment:
> The snapshot is actually of the button

Yeah i think thats because I'm snapshotting the container, should snapshot the `baseElement` instead because the popover will be mounted to the body.